src/Uno.UI/UI/Xaml/Controls/FlipView/FlipView.iOS.cs

```diff
@@ -87,6 +86,8 @@ protected override void UpdateItems()
 			}
 			base.UpdateItems();
```

Comment:
the return value of base.UpdateItems() wouldn't be appropriate?

Reply:
It would be, my bad